crates/services/gas_price_service/src/v1/service.rs

```diff
@@ -182,6 +182,13 @@ where
             storage_tx
                 .set_recorded_height(recorded_height)
                 .map_err(|err| anyhow!(err))?;
+        } else {
+            // we default to the l2 block height
+            // this is done so that we poll the da with a specific height
+            // to avoid initial loss
+            storage_tx
+                .set_recorded_height(height.into())
+                .map_err(|err| anyhow!(err))?;
```

Comment:
Do we want to add some tests around this?

Reply:
yeah sure, just wanted to make sure we don't break any other tests before writing it

Reply:
addressed in [50e0299](https://github.com/FuelLabs/fuel-core/pull/2523/commits/50e0299bc3ad4906e89c9b3fc6b00ecd1014a0e1)

Reply:
removed that, the test should be for uninitialized task: [b151207](https://github.com/FuelLabs/fuel-core/pull/2523/commits/b151207e5ba165a56e8d660b7ae0d591d9ab3980)